doc/nrf/config_and_build/bootloaders/bootloader_external_flash.rst

```diff
@@ -47,6 +47,8 @@ This error could be caused by the following issues:
 
   * An out-of-tree external flash driver is not selecting :kconfig:option:`CONFIG_PM_EXTERNAL_FLASH_HAS_DRIVER`, resulting in partitions for images located in the external flash memory being not accessible.
 
+  * Insufficient value set for ``CONFIG_BOOT_MAX_IMG_SECTORS``, as MCUboot typically increases slot sizes when external flash is enabled.
```

Comment:
Also the Kconfig CONFIG_BOOT_MAX_IMG_SECTORS is not mentioned anywhere else but here. If it requires a specific value, shouldn't this be mentioned in the configuration steps, not in troubleshooting? How is the user supposed to know about setting the value before checking this section?

Note that all other Kconfigs in this section refer to ":ref:`pm_external_flash`", except `CONFIG_PM_EXTERNAL_FLASH_HAS_DRIVER`, which is however mentioned in the same ref.
If you decide to keep the mention of ``CONFIG_BOOT_MAX_IMG_SECTORS`` in troubleshooting, I think it should also be ref'ing a page with more information.

Reply:
> Kconfig CONFIG_BOOT_MAX_IMG_SECTORS is not mentioned anywhere else but here. If it requires a specific value, shouldn't this be mentioned in the configuration steps, not in troubleshooting? How is the user supposed to know about setting the value before checking this section?

@hellesvik-nordic Is it correct that setting this Kconfig option is not **required**?

If so, then I think it can stay here. I agree though that perhaps this info should be placed somewhere else? Looking into it.

Reply:
CONFIG_BOOT_MAX_IMG_SECTORS defaults to a value used for internal flash.
However, if you enable external flash, you need to increase CONFIG_BOOT_MAX_IMG_SECTORS, or the application will never boot.